css/styles.css

```diff
@@ -188,3 +201,199 @@ Headline
   font-size: 2rem;
   padding: 0.4rem;
 }
+
+/*
+=====================================
+My works section
+=====================================
+*/
+
+.works {
+  padding: 0 var(--horizontal-spacing);
+}
+
+.works__title,
+.aboutme__title {
+  text-align: center;
+  margin-bottom: 1rem;
+}
+
+.works__title-underline {
+  border: 3px solid #36b37e;
+  background-color: #36b37e;
+  border-radius: 4px;
+  width: 50px;
+  margin-inline: auto;
+  margin-bottom: 3.5rem;
+}
+
+.works__cards {
+  display: grid;
+  gap: 1.5rem;
+  margin-bottom: 1.5rem;
+}
+
+.works__card {
+  border: 3px solid #d4dcd8;
+  border-radius: var(--border-radius);
+  overflow: hidden;
+}
+
+.card__image-container {
+  position: relative;
+}
+
+.card__image-container::before {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  aspect-ratio: 1;
+  content: "";
+  background-color: #ebf0eea7;
+  transition: background-color 400ms ease-in-out;
+}
+
+.works__card:hover > .card__image-container::before {
+  background-color: #ebf0ee20;
+}
+
+.card__image {
+  object-fit: cover;
+  height: 250px;
+}
+
+.card__content {
+  padding: 1rem;
+  border-radius: var(--border-radius);
+  background-color: #fff;
+  translate: 0 -10px;
+}
+
+.card__title {
+  color: #3a4a42;
+  font-size: 2rem;
+  text-align: center;
+}
+
+.card__technologies {
+  color: #3a4a42;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.5rem;
+  padding-left: 0;
+  list-style-type: none;
+  margin-block: 1rem;
+}
```

Comment:
- [ ] Kindly make sure the project stack list is aligned in the center of the card container in order to meet the provided Figma design.

    - ### YOUR DESIGN
      ![image](https://user-images.githubusercontent.com/87197412/205188353-8c3c8dbc-7eaa-4e12-9474-5492bf714347.png)

    - ### BETTER DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/205188532-e7de6acc-e524-45c0-beca-da18f3fae64a.png)